packages/graphql/src/registry.ts

```diff
@@ -0,0 +1,84 @@
+import {
+  GraphQLBoolean,
+  GraphQLEnumType,
+  GraphQLInputObjectType,
+  GraphQLObjectType,
+  GraphQLScalarType,
+  GraphQLUnionType,
+  type GraphQLSchemaConfig,
+} from "graphql";
+
```

Comment:
I think we might also need a `Context` to be part of the registry map. E.g. the `UsageFlags` for the model could be part of this context. Other things to consider would be visibility.

Reply:
Not sure if TypeKits can help with this